crypto/bio/bio_cb.c

```
@@ -77,6 +77,7 @@ long BIO_debug_callback(BIO *bio, int cmd, const char *argp,
 
     len = BIO_snprintf(buf,sizeof buf,"BIO[%p]: ",(void *)bio);
 
+    len = (len < 0) ? 0 : len;
```

make this "if len < 0 len = 0" with a comment about ignoring errors.